app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -34,6 +35,10 @@ keywords:
 
 Postgraduate teaching bursaries and scholarships are only available for the subjects listed below. You cannot receive both a teaching bursary and a scholarship.
 
+You cannot receive a scholarship or bursary if your teacher training course is salaried.
```

Comment:
You cannot receive a scholarship or bursary if your teacher training course is salaried. - can we say 'pays a salary' for clarity?

Reply:
We talked about salaried courses on https://getintoteaching.education.gov.uk/train-to-be-a-teacher/if-you-have-a-degree so this is consistent with other terminology on GIT

NB If we did want to change the wording Find talk about "courses that come with a salary" rather than 'pays a salary'

![image](https://user-images.githubusercontent.com/87642394/195115107-c0fef784-fe4f-4a67-a74f-c3a2fd8c8931.png)

Reply:
yep, should probably keep as 'salaried' just so that we're consistent across the site